jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -569,6 +569,15 @@ def test_default_spectral_extraction(cubeviz_helper, spectrum1d_cube_fluxunit_jy
         extracted_spectra[0].flux, extracted_spectra[1].flux
     )
 
+    # Also test unit conversion
+    uc = cubeviz_helper.plugins["Unit Conversion"]._obj
+    uc.flux_unit_selected = "MJy"
```

Comment:
maybe just make a note that the original units should _not_ be MJy for the test to pass (or include an assert for that)?

Reply:
Assert added. Thanks!